docs/Files/UI/Solution-Manager/Library/ServerOptions/Managing-SSO-Configurations.md

```diff
@@ -0,0 +1,207 @@
+    1.  Inside the modal, select the following options: 1. Sign-in method: OIDC - OpenID Connect 1. Application type: Single-Page Application
+    - ![Okta-Application](../../../../../Resources/Images/SM/Library/ServerOptions/okta-app-integration.png "Okta - Integration Options")
+    1. Click on "Next"
+    1. Set the following options:
+       1. Grant type: Authorization
+       1. Sign-in redirects URIs:(sample) `https://<hostaname:443>/login/callback`
+          1. The **hostname** will be used in the SSO Configuration page in Solution Manager. Make sure to include the **port number** if applicable
+       1. The value for URL to access Solution Manager will be used in the configuration settings
+       1. Sign-out redirects URIs: This value is not necessary
+       1. Controlled access: Allow everyone in your organization access (for this example)
+       1. If an option was not listed, you may select the one that is more convenient for your organization
+       1. Set controlled access based on your organizations needs
+       - ![Okta-Application](../../../../../Resources/Images/SM/Library/ServerOptions/okta-newpageapp.png "Okta- Integration Grant Type")
+    1. Click on "Save".
+    1. Note down the “Client ID” value, this will be used to configure SSO in solution manager
+1.  After creating the application, go the newly created application and go to "Assignments"
+    1. Assign the users from your organization to the new application
+    - ![Okta-Application](../../../../../Resources/Images/SM/Library/ServerOptions/okta-user-assignments.png "Okta- User Assignments")
+1.  Go to the "Security" section in the main menu and select "API"
+    1. In the "Authorization Server" tab select the "default" server
+       1. Note down the Issuer URI for the “default” row, this value will be the “authority” which will be used in the configuration screen for SSO
+    1. Inside the "default" authorization server select the "Claims" tab
+    - ![Okta-Application](../../../../../Resources/Images/SM/Library/ServerOptions/okta-default-server.png "Okta - Claims Tab")
+    1. Click on "Add Claim"
+       1. The following is an example on how to add a custom claim
+       - ![Okta-Application](../../../../../Resources/Images/SM/Library/ServerOptions/okta-group-claim.png "Okta - Add Group Claim")
+       1. Make sure the token type is Access Token and that you enter **`.*`** for the regex logic
+       1. Make sure to name claims “**groups**”
+       1. Repeat the process to add an “**email**” claim
+          1. Make sure to add **appuser.email** in the **Value** text field
+       - ![Okta-Application](../../../../../Resources/Images/SM/Library/ServerOptions/okta-claim-email.png "Okta - Add Email Claim")
+    1. Click on the “Settings” tab and take notes of the following values:
+       1. Audience will be used in the configuration settings page inside Solution Manager
```

Comment:
1. Audience, this will be used in the configuration settings page inside Solution Manager